reporting.md

```diff
@@ -114,11 +114,9 @@ We also define the following terms for further use in this document:
 
 * This document is intended as a companion to the specification for SMTP MTA
     Strict Transport Security (MTA-STS, TODO: Add ref).
-* The Public Key Pinning Extension for HTTP [@!RFC7469] contains a JSON-based
-    definition for reporting individual pin validation failures.
-* The Domain-based Message Authentication, Reporting, and Conformance (DMARC)
```

Comment:
Hmm, OK. I think it would be nice to reference them somewhere just so people can get a sense of similar "reporting" technologies. But I agree they aren't critical here.